release/release.go

```diff
@@ -459,6 +555,15 @@ func copyFile(src, dst string) error {
 	return out.Close()
 }
 
+func getRPMVersion(version string) string {
+	// r49.3.2-39-g7f57f9a2 will be turned to 49.3.2
```

Comment:
For deb, we didn't strip out the pre-release; does rpm not support any thing other than MAJOR.MINOR.PATCH?

Reply:
Let me double check.

Reply:
prerelease goes in the "Release" category rather than the version.